json.h

```diff
@@ -0,0 +1,75 @@
+#ifndef json_h
+#define json_h
```

Comment:
hah, you're right - I looked at the first other header I saw in this project, `utlist.h`, without thinking (it's more of a copied-in library, one I've used before, so I should know)